src/scripts/Fetchers/LemmyFetcher.js

```diff
@@ -15,12 +15,10 @@ export class LemmyFetcher extends Fetcher {
                 throw new Error(`HTTP error: ${response.status}`);
             }
             let response_data = await response.json();
-            let posts = []
             const posts_json = response_data.posts
             posts_json.forEach(post => {
-                posts.push(this.#createPost(post))
+                this.container.appendChild(this.#createPost(post))
```

Comment:
This function would no longer return an array of posts. Can you update the docs as well ? Thx.

Reply:
Sure, thanks for mentioning it